docs/README.adoc

```diff
@@ -16,14 +16,21 @@ toc::[levels=1]
 
 == templates
 
-* link:./templates/eap-cd-amq-persistent-s2i.adoc[eap-cd-amq-persistent-s2i]
-* link:./templates/eap-cd-amq-s2i.adoc[eap-cd-amq-s2i]
-* link:./templates/eap-cd-basic-s2i.adoc[eap-cd-basic-s2i]
-* link:./templates/eap-cd-https-s2i.adoc[eap-cd-https-s2i]
-* link:./templates/eap-cd-sso-s2i.adoc[eap-cd-sso-s2i]
-* link:./templates/eap-cd-starter-s2i.adoc[eap-cd-starter-s2i]
-* link:./templates/eap-cd-third-party-db-s2i.adoc[eap-cd-third-party-db-s2i]
-* link:./templates/eap-cd-tx-recovery-s2i.adoc[eap-cd-tx-recovery-s2i]
+* link:./templates/eap73-amq-persistent-s2i.adoc[eap73-amq-persistent-s2i]
+* link:./templates/eap73-amq-s2i.adoc[eap73-amq-s2i]
+* link:./templates/eap73-basic-s2i.adoc[eap73-basic-s2i]
+* link:./templates/eap73-https-s2i.adoc[eap73-https-s2i]
+* link:./templates/eap73-openjdk11-amq-persistent-s2i.adoc[eap73-openjdk11-amq-persistent-s2i]
+* link:./templates/eap73-openjdk11-amq-s2i.adoc[eap73-openjdk11-amq-s2i]
+* link:./templates/eap73-openjdk11-basic-s2i.adoc[eap73-openjdk11-basic-s2i]
+* link:./templates/eap73-openjdk11-https-s2i.adoc[eap73-openjdk11-https-s2i]
+* link:./templates/eap73-openjdk11-sso-s2i.adoc[eap73-openjdk11-sso-s2i]
+* link:./templates/eap73-openjdk11-third-party-db-s2i.adoc[eap73-openjdk11-third-party-db-s2i]
+* link:./templates/eap73-openjdk11-tx-recovery-s2i.adoc[eap73-openjdk11-tx-recovery-s2i]
+* link:./templates/eap73-sso-s2i.adoc[eap73-sso-s2i]
+* link:./templates/eap73-third-party-db-s2i.adoc[eap73-third-party-db-s2i]
+* link:./templates/eap73-tx-recovery-s2i.adoc[eap73-tx-recovery-s2i]
+* link:./templates/fn.adoc[fn]
```

Comment:
is the footnote (I presume) link on purpose?

Reply:
Good question, never noticed that before, I'll have to look.